css/about-me.css

```diff
@@ -63,6 +63,12 @@
   color: #fff;
   border-style: none;
   border-radius: 4px;
+  transition: all 1s;
+}
+
+.get-my-resume-btn:hover {
+  background: #008552;
+  transform: translateY(-20px);
 }
 
 @media screen and (min-width: 768px) {
```

Comment:
Kindly check the position, size, margins and paddings of the about me section. The image to the left and right of the about me description seems to close compared to the actual design. Kindly check and fix this anomalies. 

### Your design 

![Screenshot from 2023-02-25 12-05-12](https://user-images.githubusercontent.com/65233860/221356514-6a85c141-6abf-40ea-8ecb-b39f8ecc8b24.png)



### Actual design 


![Screenshot from 2023-02-25 12-07-01](https://user-images.githubusercontent.com/65233860/221356516-694ac79a-6b6d-4efd-8d55-063ee47d27cc.png)